### ISSUE TITLE
The view button on github is overlaid on the site header

### ISSUE THREAD

**Issue by Franqsanz** [forem/forem#17183]
Hello forem team I have discovered the following problem in the interface.
The view button on github is overlaid on the site header, the solution is to remove the  `position: relative`

Example below:

![1](https://user-images.githubusercontent.com/44296203/162450607-d62b98fc-d346-4dde-92d2-eefcba43addf.png)

At the time of scrolling the following happens:

![2](https://user-images.githubusercontent.com/44296203/162450258-aca9b243-bcb7-4341-9676-a0f3f0353d1a.png)

Solution:

![3](https://user-images.githubusercontent.com/44296203/162450261-bd4abd04-d279-43a1-bc1d-551919d3081c.png)

Thanks!!

**Comment by tnir** [comment]
@Franqsanz Good catch! `position: static` may resolve this problem, but transparency above the button will happen as follow:

| as is | `position: relative` removed (`position: static`) |
|-|-|
| ![position-relative](https://user-images.githubusercontent.com/10229505/162560695-a038fc31-6a80-43f6-9cc3-dfc3c9428b99.png) | ![position-static](https://user-images.githubusercontent.com/10229505/162560692-adfc13b8-a09d-4034-b98c-e7285e005013.png) |

**Comment by tnir** [comment]
Note: While #17196 was merged, that was not rolled out into production.

**Comment by tnir** [comment]
@Franqsanz I confirmed as resolved in production just now, so let me know or open a new issue if you find a problem 👍

**Comment by tnir** [comment]
@ludwiczakpawel Thanks for following up. I could not find information when to deliver the merged commit to the production e in https://developers.forem.com/ . Is any commit on main designed to be delivered to production (without any heuristic process or periodic delays)?

**Comment by ludwiczakpawel** [comment]
@tnir AFAIK every merge is being deployed to dev.to "right away" (usually it needs a bit of time to build etc).
